scripts/variantstore/scripts/cromwell_monitoring_script.sh

```diff
@@ -0,0 +1,297 @@
+#!/bin/bash
+# NOTE: this script is intended to be placed in google cloud storage
+# and invoked by adding the following line to your cromwell workflow
+# options:
+#    "monitoring_script": "gs://bucket/path/to/cromwell_monitoring_script.sh"
+# Upon task completion "monitoring.log" will be added to the appropriate
+# cloud storage folder.
+set -Eeuo pipefail
+
+if [[ -e /mnt/disks/cromwell_root ]]
+then
+  MONITOR_MOUNT_POINT_DEFAULT=/mnt/disks/cromwell_root
+else
+  MONITOR_MOUNT_POINT_DEFAULT=/cromwell_root
+fi
+
+MONITOR_MOUNT_POINT=${MONITOR_MOUNT_POINT:-$MONITOR_MOUNT_POINT_DEFAULT}
```

Comment:
The only changes I made in this script are here, to allow for the possibility of different mount points.